include/dem/dem.h

```diff
@@ -360,8 +359,23 @@ class DEMSolver
   std::vector<Tensor<1, 3>> force;
   std::vector<double>       displacement;
   std::vector<double>       MOI;
-  Tensor<1, dim>            periodic_offset;
-  bool                      has_periodic_boundaries;
+
+  // Mesh and boundary information
+  typename dem_data_structures<dim>::floating_mesh_information
+    floating_mesh_info;
+  typename dem_data_structures<dim>::boundary_points_and_normal_vectors
+    updated_boundary_points_and_normal_vectors;
+  typename dem_data_structures<dim>::vector_on_boundary
+    forces_boundary_information;
+  typename dem_data_structures<dim>::vector_on_boundary
+    torques_boundary_information;
+  typename DEM::dem_data_structures<dim>::periodic_boundaries_cells_info
+    periodic_boundaries_cells_information;
```

Comment:
Should these types be defined here or could they be defined in another .h file to be more neutral?

Reply:
Mhm I don't know, they are not really related to each other, it's again a bit random to me. What do you think?

Reply:
Good for me then :)!